charts/m4d/templates/secure-control-plane.yaml

```diff
@@ -0,0 +1,43 @@
+{{- if .Values.secureControlPlane }}
```

Comment:
The Istio resources where going to be added in https://github.com/IBM/the-mesh-for-data/pull/450 but then I decided to drop them in favor of matching documentation in the website for now. I suggest to drop and re-open the other PR if we think it should be added (and decide how best to add them).